modules/ximgproc/src/structured_edge_detection.cpp

```diff
@@ -419,7 +419,16 @@ class StructuredEdgeDetectionImpl : public StructuredEdgeDetection
     /*!
      * This constructor loads __rf model from filename
      *
-     * \param filename : name of the file where the model is stored
+     * Refer to <https://github.com/opencv/opencv_extra/blob/master/testdata/cv/ximgproc/model.yml.gz>
```

Comment:
This description should be visible from the generated documentation, so it's better to place it 
before `createStructuredEdgeDetection`. And also `create` functions are usually satic methods of corresponding classes (template methods of Algorithm class supposes that). Could you please move `createStructuredEdgeDetection` inside `StructuredEdgeDetection` and rename it to `create`?